src/xdocs/config_filters.xml

```diff
@@ -648,6 +648,220 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathFilter">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.4</p>
+        <p>
+          Filter <code>SuppressionXpathFilter</code> works as
+          <code>SuppressionFilter</code>.Additionally, filter
+          processes <code>suppress-xpath</code> elements,
+          which contains xpath-expressions. Xpath-expressions
+          are queries for suppressed nodes inside the AST tree.
+        </p>
```

Comment:
please mention in new P tag, that CLI have options to print Xpath, to ease usage by user.

Reply:
CLI option will be available after resolving (4530)[https://github.com/checkstyle/checkstyle/issues/4530], should I mention this issue in `config_filters.html`?

Reply:
please do, as reminder to not forget.... I already forgot.

Reply:
done